src/Components/ServiceNotConfigured/ServiceNotConfigured.js

```diff
@@ -8,8 +8,12 @@ export const ServiceNotConfigured = () => (
         <EmptyState className='empty-state-not-configured'>
             <EmptyStateIcon icon={WrenchIcon} />
             <Title headingLevel="h5" size="lg">
-                Resource optimization is not yet configured
+                Resource optimization isn&apos;t configured yet
             </Title>
+            <EmptyStateBody>
+                To start configuration, activate this application in the Service Enabled Dashboard.
+                <Text>[Start configuration]</Text>
```

Comment:
Hey @apuntamb, do not merge this pull-request immediately as I would like to know - whether `[start configuration]` is just a placeholder or any action expected here?
I will confirm this behaviour with rest of the team.

Reply:
As per offline discussion, initially ROS will provide PDF link for end-user. 
While GA, needs to link official doc link at [Start configuration].
For now, both are not available so keeping this PR on hold.